provisioning/stream/kubernetes/templates/api/deployment.yaml

```diff
@@ -92,6 +92,8 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.labels['tags.datadoghq.com/version']
+            - name: OTEL_METRICS_EXEMPLAR_FILTER
+              value: "always_off"
```

Comment:
I found a similar issue here: https://github.com/open-telemetry/opentelemetry-go/issues/5936

Couldn't find a good place where to call `sdkmetric.WithExemplarFilter(exemplar.AlwaysOffFilter)` in our code since we don't use `sdkmetric` directly anywhere but I found [this function](https://github.com/open-telemetry/opentelemetry-go/blob/ac5caeaee88d744b21dc3789ebda79ae97f6cfa5/sdk/metric/config.go#L158-L172) which suggests that we can disable exemplars using this env variable.